etc/profile-m-z/skypeforlinux.profile

```diff
@@ -15,15 +15,22 @@ ignore nou2f
 ignore novideo
 ignore private-dev
 ignore dbus-user none
-ignore dbus-system none
 
 # breaks Skype
 ignore apparmor
 ignore noexec /tmp
 
 noblacklist ${HOME}/.config/skypeforlinux
 
+mkdir ${HOME}/.config/skypeforlinux
+whitelist ${HOME}/.config/skypeforlinux
+
 # private-dev - needs /dev/disk
 
+dbus-user filter
+dbus-user.talk org.freedesktop.Notifications
+dbus-user.talk org.freedesktop.secrets
+dbus-user.talk org.kde.StatusNotifierWatcher
```

Comment:
```suggestion
?ALLOW_TRAY: dbus-user.talk org.kde.StatusNotifierWatcher
```

Reply:
OT: We should add a profile check for this.

Reply:
FYI: Skype will immediately log you out if this is not present, so I might be worth leaving a comment....?

Reply:
In that case, I would say leave it enabled but add a comment that we do not add `?ALLOW_TRAY:` later on profile clean ups.

Reply:
Oh, I've already added it. Given that it's enabled by default it sounds kinda safe...? 
But let me know your preference, either way sounds okay to me.

Reply:
> Given that it's enabled by default it sounds kinda safe...?

What? allow-tray or StatusNotifierWatcher?

Reply:
>  But let me know your preference, either way sounds okay to me (I don't use a tray anyway).

If skype does not work without `dbus-user.talk org.kde.StatusNotifierWatcher`, we should allow it.